_data/variables.yml

```diff
@@ -850,132 +850,148 @@
   contents:
     - name: sm
       subsection: border-radius token
-      description: Sets `'sm'` `border-radius` theme token
+      description: Sets `'sm'` `border-radius` theme token.
```

Comment:
I won't mark this every time, but another reminder to use double quotes in examples instead of single quotes. I recognize that this is a BIG issue across the repo (thank Prettier!), but let's start with consistency on this page